spec/views/collections/_show_document_list_menu.html.erb_spec.rb

```diff
@@ -1,18 +1,35 @@
 
 describe 'collections/_show_document_list_menu.html.erb', type: :view do
-  context 'when user logged in displaying the collections of current user' do
+  context 'when user is viewing a collection' do
```

Comment:
I think this drop down menu is generated in the action column of a row for an individual work on a single collection index page unless it is also used in additional contexts that I haven't seen.